json-rpc/docs/type_account.md

```diff
@@ -17,11 +17,11 @@ A Libra account.
 | sent_events_key                   | string                         | Unique key for the sent events stream of this account                                       |
 | received_events_key               | string                         | Unique key for the received events stream of this account                                   |
 | is_frozen                         | boolean                        | Whether this account is frozen or not                                                       |
-| role                              | string or object               | One of the following values:                                                                |
-|                                   |                                |   - "unknown"                                                                               |
-|                                   |                                |   - { "designated_dealer" => [DesignatedDealerRole](#type_designateddealerrole) }           |
-|                                   |                                |   - { "parent_vasp" => [ParentVASPRole](#type_parent_vasp) }                                |
-|                                   |                                |   - { "child_vasp" => [ChildVASPRole](#type_childvasprole) }                                |
+| role                              | object                         | One of the following type:                                                                |
+|                                   |                                |   - [UnknownRole](#type_unknownrole) |
+|                                   |                                |   - [ParentVASPRole](#type_parentvasprole) |
+|                                   |                                |   - [ChildVASPRole](#type_childvasprole) |
+|                                   |                                |   - [DesignatedDealerRole](#type_designateddealerrole) |
```

Comment:
it was my mistake, the underscore after type "_" should be dash "-"

Reply:
yeah, fixed.